Specs/Scene/GlobeSurfaceTileProviderSpec.js

```diff
@@ -573,13 +573,13 @@ defineSuite([
         switchViewMode(SceneMode.SCENE3D, new GeographicProjection(Ellipsoid.WGS84));
 
         return updateUntilDone(scene.globe).then(function() {
-            expect(scene).toRender([0, 0, 127, 255]); // default baseColor
+            expect(scene).toRender([0, 0, 128, 255]); // default baseColor
```

Comment:
This test is failing for me. The result is 127, not 128.

Reply:
Hmmm, I swear it was passing on Windows/Linux on my Intel system, but now I'm seeing the same thing on Windows + Nvidia. Either way, poorly designed spec...